### DISCUSSION TITLE
How do I display ampersand in flowchart markdown?

### DISCUSSION THREAD

**Discussion by dralangrant** [mermaid-js/mermaid#5821] [Q&A]
I'm trying to display an ampersand in a multiline markdown text flowchart node. I keep seeing the HTML entity name display no matter which ampersand variation I try.

I've tried them all and still only see &amp; no matter what I use. Any fixes or ways to escape so it will display in the markdown flowchart node?

**Comment by ellafm** [comment]
Can you provide the code that you are using which causes the error?
